products/microos/main.pm

```diff
@@ -58,6 +58,7 @@ sub load_installation_tests {
         loadtest('installation/installation_mode');
         loadtest('installation/logpackages');
         loadtest('installation/system_role');
+        loadtest "installation/ntp_config_settings";
```

Comment:
Works, but the inconsistent function call and quoting syntax makes this stand out...

Reply:
yeah sorry - copy pasted that from the other main.pm...